### DISCUSSION TITLE
HW 4 Part 3 Question 2

### DISCUSSION THREAD

**Discussion by ecieszynski** [cooperrc/computational-mechanics#61] [Q&A]
Not sure what is going on with this code. I know it isn't fitting well because I am not sure how to use 50 points in time to get the smooth best fit line, but I feel like that isn't the only issue going on here

![image](https://user-images.githubusercontent.com/106542680/175839439-57858547-9bb6-456f-b65d-81e80ec8838b.png)

**Comment by cooperrc** [comment]
```python
Z = np.block[[x**2], [x], [x**0]]).T
```

This is perfect for a quadratic function, $y(x) = Ax^2+Bx+C$, which is exactly what was plotted in the image. 

the problem is that we want exponential functions, 

```python
Z = np.block([[np.exp(-x/1.71)], [np.exp(-x/11)], ...])
```
